package.json

```
@@ -66,7 +73,7 @@
     "lerna": "3.22.1",
     "lint-staged": "^10.0.1",
     "mocha": "^8.0.1",
-    "prettier": "2.0.5",
+    "prettier": "^2.0.5",
```

We pin the version of prettier we use, to avoid surprise formatting changes if version get upgraded by dependabot or while running yarn upgrade
```suggestion
    "prettier": "2.0.5",
```